controller/RestController.php

```diff
@@ -1063,4 +1063,53 @@ public function related($request)
         $ret = $this->transformPropertyResults($request->getUri(), $request->getLang(), $related, "related", "skos:related");
         return $this->returnJson($ret);
     }
+
+    /**
+     * Used for querying new concepts in the vocabulary
+     * @param Request $request
+     * @return object json-ld wrapped list of changed concepts
+     */
+    public function newConcepts($request)
+    {
+        return $this->changed($request);
+    }
+
+    /**
+     * Used for querying modified concepts in the vocabulary
+     * @param Request $request
+     * @return object json-ld wrapped list of changed concepts
+     */
+    public function modifiedConcepts($request)
+    {
+        return $this->changed($request, 'dc:modified');
+    }
+
+    /**
+     * Used for querying changed concepts in the vocabulary
+     * @param Request $request
+     * @return object json-ld wrapped list of changed concepts
+     */
+    private function changed($request, $prop='dc:created')
```

Comment:
I think it would better to omit the `$prop` default value here and instead pass the property it explicitly from both `newConcepts` and `modifiedConcepts`. That would be more symmetrical.

The parameter `$prop` is missing a docstring definition.

Minor style comment: renaming the function to `changedConcepts` would make it better in line with `newConcepts` and `modifiedConcepts`.

Reply:
Good point, I made the changes you suggest.